### PR TITLE
fix: export feature and update packages

### LINKED ISSUE #41: [bug] can't export in excalidraw
image

Clicking the "Export" button causes an error.

image

### PR DESCRIPTION
**Pull request by dai-shi**
fix #41.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
   document.getElementById("root")
 );
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+// HACK for https://github.com/dai-shi/excalidraw-claymate/issues/41
+(window as any).React = { createElement };
```
Comment (dai-shi) [inline src/index.tsx:20]: This is probably only necessary to fix #41, but let's keep others as various things are updated.

Comment (grant-davidson) [reply src/index.tsx:20]: I understand all the changes except this one. What does this do?

Comment (dai-shi) [reply src/index.tsx:20]: Without this, #41 bug persists. Somehow, `@excalidraw/excalidraw` package can't find `React.createElement` in the production build.
If you find another solution, much appreciated.

Comment (grant-davidson) [reply src/index.tsx:20]: Are you able to reproduce #41 locally?  I built claymate and ran it locally but can't reproduce the problem on master

Comment (dai-shi) [reply src/index.tsx:20]: Yes, this is reproducible for me locally.

Comment (grant-davidson) [reply src/index.tsx:20]: I still can't reproduce the problem and the window element never seems to have a React property so I can't really add anything useful to this review.

**Review by grant-davidson — APPROVED** [review]

File: src/index.tsx (hunk 1 above)
Comment (dai-shi) [reply src/index.tsx:20]: really... does the problem repro in the production for you?
(I don't like this hack and wish we have a proper fix.)

Comment (grant-davidson) [reply src/index.tsx:20]: No, I presume all I need to do is:
Open: https://dai-shi.github.io/excalidraw-claymate/
Click: "Export Gif"
Enter a file name & click "Save"

Comment (grant-davidson) [reply src/index.tsx:20]: wBV7l3zsLH

Comment (dai-shi) [reply src/index.tsx:20]: No, it's "Export" in excalidraw.
image

Comment (grant-davidson) [reply src/index.tsx:20]: Ok, now I see the problem!!!

**Comment by dai-shi** [comment]
Let me merge this for now. It might be a upstream issue, and there might be a better workaround. Feel free to suggest improvements.

**Pull request merged**